e2e/playwright/info-drawer/src/tests/comments.spec.ts

```diff
@@ -0,0 +1,153 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test, TrashcanApi, FavoritesPageApi, NodesApi, FileActionsApi } from '@alfresco/playwright-shared';
+
+test.describe('Info Drawer - Comments', () => {
+  const apiClientFactory = new ApiClientFactory();
+  let nodesApi: NodesApi;
+  let trashcanApi: TrashcanApi;
+  let fileActionsApi: FileActionsApi;
+  let favoritesActions: FavoritesPageApi;
+  const username = `user-e2e-${Utils.random()}`;
+  const commentText = `comment-e2e-${Utils.random()}`;
+  const persFolderName = `persFolder-e2e-${Utils.random()}`;
+  const favFolderName = `favFolder-e2e-${Utils.random()}`;
+  const sharedFileName = `sharedFile-e2e-${Utils.random()}`;
+
+  test.beforeAll(async () => {
+    try {
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username });
+      nodesApi = await NodesApi.initialize(username, username);
+      trashcanApi = await TrashcanApi.initialize(username, username);
+      favoritesActions = await FavoritesPageApi.initialize(username, username);
+      fileActionsApi = await FileActionsApi.initialize(username, username);
+    } catch (error) {
+      console.error(`beforeAll failed : ${error}`);
+    }
+  });
+
+  test.beforeEach(async ({ loginPage }) => {
+    try {
+      await loginPage.loginUser({ username: username, password: username }, { withNavigation: true, waitForLoading: true });
```

Comment:
Small thing but object inside loginUser can be just: 
{ username, password: username} - no need to repeat username at right side when left and right sides are the same

Reply:
Good to know, thanks! Will be fixed in the next push